app/services/notifications/new_endorsement/send.rb

```diff
@@ -0,0 +1,77 @@
+# send notifications about the new comment
+
+module Notifications
+  module NewEndorsement
+    class Send
+      def initialize(endorsement)
+        @endorsement = endorsement
+      end
+
+      delegate :user_data, :endorsement_data, to: Notifications
+
+      def self.call(*args)
+        new(*args).call
+      end
+
+      def call
+        json_data = {
+          user: user_data(endorsement.user),
+          endorsement: endorsement_data(endorsement)
+        }
+
+        targets = []
+        user_id = 11
+        Notification.create(
+          user_id: user_id,
+          notifiable_id: endorsement.id,
+          notifiable_type: endorsement.class.name,
+          action: nil,
+          json_data: json_data,
+        )
+
+        targets << "user-notifications-#{user_id}" if User.find_by(id: user_id)
+
+        # Sends the push notification to Pusher Beams channels. Batch is in place to respect Pusher 100 channel limit.
+        targets.each_slice(100) { |batch| send_push_notifications(batch) }
+      end
+
+      private
+
+      attr_reader :endorsement
+
+      def send_push_notifications(channels)
```

Comment:
maybe we can factor this out from `app/services/notifications/new_comment/send.rb`